vagrant/ansible/roles/ctdb.setup/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Create the ctdb replicated volume
+  gluster_volume:
+    state: present
+    name: "{{ gluster_features_ctdb_volume }}"
+    bricks: '{{ ctdb_brick_location }}/ctdb'
+    cluster: "{{ gluster_cluster_hosts }}"
+    replicas: "{{ gluster_cluster_replica_count }}"
+  run_once: yes
+
+- name: Add ctdb package
+  yum:
+    name: ctdb
+    state: present
+
+- name: Add ctdb rules to firewalld
+  firewalld:
+    service: ctdb
+    permanent: yes
+    state: present
+    state: enabled
+
+- name: Add recovery lock to ctdb.conf
+  lineinfile:
+    dest: /etc/ctdb/ctdb.conf
+    line: "\trecovery lock = /gluster/lock/recovery.lock"
+    regexp: "recovery lock"
+    insertafter: '\[cluster\]'
+
+- name: Add netbios name - The two ctdb nodes should have the same netbios name
+  lineinfile:
+    dest: /etc/samba/smb.conf
+    line: "netbios name = {{ ctdb_netbios_name }}"
+    regexp: "netbios name"
+    insertafter: '\[global\]'
+
+
+# Check bz 1656777
+- name: Create missing directories for ctdb package
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: root
+  with_items:
+    - /etc/ctdb/events/legacy
+    - /var/lib/ctdb/state
+    - /var/lib/ctdb/persistent/
+    - /var/lib/ctdb/volatile/
+
+- name: Apply SELinux file context to new directories
+  command: restorecon -irv /etc/ctdb/events/legacy /var/lib/ctdb/state
+
+#Temporary
+- name: Put SELinux in permissive mode, logging actions that would be blocked.
+  selinux:
+    policy: targeted
+    state: permissive
+
+- include_role:
+    name: gluster.features
+
+- name: Enable check consistency of databases during startup
+  command: '/bin/ctdb event script enable legacy 00.ctdb'
+
```

Comment:
`01.reclock` is also preferred to be enabled.

Reply:
Why do we need to enable these?
Aren't they enabled by the package?

Reply:
Anoop, done. Checked the RHGS samba package scripts and that was the only one missing.
Obnox, Centos7 ctdb packaging issues. I've used a block to setup workarounds for the package. This will go once packaging issues have been resolved.